### ISSUE TITLE
write joburg trip ad series ad script

### ISSUE THREAD

**Issue by CIEspost** [zacck/cieproductions#31]
use the places we stop at along the way

See script here

https://docs.google.com/spreadsheets/d/1mAAe3GMXWNJjFnLyb9SObhSE_54zor8jGSy59DOZspU/edit#gid=0

**Comment by zacck-zz** [comment]
What does this task need ?

**Comment by CIEspost** [comment]
Dont be Hungry
Be Hapi

Dont be Thirsty 
Be Hapi

**Comment by CIEspost** [comment]
Person sitting at campsite and then person sitting with friends in restaurant having special @zacck do you think you can do this for us as in you will always be the actor? Maybe Nadine can also be the person sometimes?

But mainly I would like to find out if you think this idea would work logistically for us, because that would mean we need to get footage of us 3 sitting at a table eating something aka specials? What would be a cheap way of doing this you think?

**Comment by CIEspost** [comment]
write same format type script as animals series because we will be making ads like documentaries on this trip it is important to see what will be best for this doccie type production possibilities

**Comment by CIEspost** [comment]
- [ ] we need to do research on how many videos we need in this series

**Comment by CIEspost** [comment]
The goal in crafting a perfect product video is not far off. If you want your video to resonate, it should be about more than just the product. It should be about the problem, the solution, the experience, and the larger vision of what you're trying to build.

**Comment by CIEspost** [comment]
The success of this video comes down to two things: a clear, unwavering message and a company with a personality.

**Comment by CIEspost** [comment]
The videos are minimally expensive, product-focused,

**Comment by CIEspost** [comment]
what's singular about this product launch video is how well the company knows its audience and the problem it's trying to solve.

**Comment by CIEspost** [comment]
The best product videos focus not on the product itself, but on the stories of the people who use it.

**Comment by CIEspost** [comment]
the pervasiveness of communications tools in our lives. In it, he explains that technology doesn't truly get interesting until it becomes so ingrained in our lives it turns invisible. No product video shows this "invisibility" of really good products better than Google's "The Web is What You Make It" series.

**Comment by CIEspost** [comment]
It's a video about experience, not software, and that is arguably what the company truly creates.

**Comment by CIEspost** [comment]
I want to end this list with a bit of an anomaly, because it pushes at the boundaries of what can be considered a product video and, as such, opens up all sorts of opportunities.

InVision, a prototyping, collaboration, and workflow platform wants to empower designers -- their primary users. Much of their content strategy is bent on this mission. This year, InVision will launch a documentary on the role of design in the modern business.

Design Disruptors looks at how 15 top businesses prioritize design in their products and overall user experience. Unlike traditional product videos, Design Disruptors will run in theaters and on Netflix. And unlike traditional product videos, Design Disruptors never actually promotes the product. The goal is bigger than the product.

**Comment by zacck-zz** [comment]
Ah then we can do some of this targeting our users and clients separately.

**Comment by CIEspost** [comment]
Yes exactly what I thought, @zacck lets start writing some ideas somewhere about this type of content

**Comment by CIEspost** [comment]
@zacck i am thinking, why dont we take the videos we have now and try and get seperate ads from them, and instead if using only 1 video, we will have various ones and they will all be much shorter (15 secs) 
- i think these a should be done asap 
this is going to be a range that markets to users specifically?

**Comment by zacck-zz** [comment]
I see lets plan how to break this down and approach it
